src/utils/gitignore.js

```diff
@@ -15,19 +15,19 @@ const parser = function (input, fn = (line) => line) {
   let section = { name: 'default', patterns: [] }
   const state = { patterns: [], sections: [section] }
 
-  for (const line of lines) {
+  lines.forEach((line) => {
```

Comment:
I find the previous code easier to understand (`continue` vs `return`).

Also, looks like we only use `ensureNetlifyIgnore` from this file, so we can delete `parser`, `parse` and `format`.

Reply:
Done.